src/main.c

```diff
@@ -39,7 +40,8 @@
 #define WORKER_BUSY 1
 #define WORKER_DONE 2
 
-typedef struct {
+typedef struct
```

Comment:
Can you reformat the code and bring back open curly braces to its original position as it just simply increased number of changes in the file which is unnecessary.

Reply:
That should be fixed now.